tools/kubernetes/conf/conf.go

```diff
@@ -132,15 +135,22 @@ func (c *config) initLog() bool {
 		DisableColors: true,
 		FullTimestamp: true,
 	})
-
-	logFile, err := os.OpenFile(c.LogPath, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
-	if err == nil {
-		Log.SetOutput(logFile)
-		return true
-	} else {
-		Log.Infof("Failed to log to file, using default stderr.")
-		return false
+	if c.FileLog {
+		logFile, err := os.OpenFile(c.LogPath, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
+		if err != nil {
+			fmt.Println("Failed to init log file settings..." + err.Error())
+			Log.Infof("Failed to log to file, using default stderr.")
+			return false
+		} else if c.ConsoleLog {
+			mw := io.MultiWriter(os.Stdout, logFile)
+			Log.SetOutput(mw)
+		} else if !c.ConsoleLog {
+			Log.SetOutput(logFile)
+		}
+	} else if c.ConsoleLog {
```

Comment:
fileLog & consoleLog seems mutual exclusive. But they should can be existed together